tests/generic_config_updater/conftest.py

```diff
@@ -28,6 +28,9 @@ def ignore_expected_loganalyzer_exceptions(duthost, loganalyzer):
              ".*ERR swss[0-9]*#orchagent: :- getPortOperSpeed.*", # test_portchannel_interface replace mtu
              ".*ERR.*Failed to apply Json change.*", # validator need updater submodule
              ".*ERR GenericConfigUpdater: Change Applier: service invoked.*", # validator need updater submodule
+             ".*getResAvailableCounters.*", # test_monitor_config
+             ".*objectTypeGetAvailability.*", # test_monitor_config
+             ".*ERR dhcp_relay.*", # test_dhcp_relay
```

Comment:
The pattern is too generic. Can you prepend the module generated these messages?

BTW, can you also change the scope of this fixture to `module` level?

Reply:
I've update the first two pattern. The dhcp_relay ERR seems random failure. The msg looks like below. So I just wrap them into one.
['Apr 11 17:53:27.057936 vlab-01 ERR dhcp_relay#dhcrelay[38]: setsockopt: SO_BINDTODEVICE: No such device\n', 
'Apr 11 17:53:27.057936 vlab-01 ERR dhcp_relay#dhcrelay[38]: \n', 
'Apr 11 17:53:27.057948 vlab-01 ERR dhcp_relay#dhcrelay[38]: If you think you have received this message due to a bug rather\n', 
'Apr 11 17:53:27.057948 vlab-01 ERR dhcp_relay#dhcrelay[38]: than a configuration issue please read the section on submitting\n', 
'Apr 11 17:53:27.057948 vlab-01 ERR dhcp_relay#dhcrelay[38]: bugs on either our web page at www.isc.org or in the README file\n', 
'Apr 11 17:53:27.057962 vlab-01 ERR dhcp_relay#dhcrelay[38]: before submitting a bug.  These pages explain the proper\n', 
'Apr 11 17:53:27.057962 vlab-01 ERR dhcp_relay#dhcrelay[38]: process and the information we find helpful for debugging.\n', 
'Apr 11 17:53:27.057973 vlab-01 ERR dhcp_relay#dhcrelay[38]: \n', 
'Apr 11 17:53:27.057973 vlab-01 ERR dhcp_relay#dhcrelay[38]: exiting.\n'

Reply:
Looks like loganalyzer fixture is function level.